buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -85,80 +85,28 @@
-        "nugetConfigPath": "",
-        "externalEndpoints": "",
-        "noCache": "false",
-        "packagesDirectory": "",
-        "verbosityRestore": "Detailed",
-        "searchPatternPush": "$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg",
-        "nuGetFeedType": "internal",
-        "feedPublish": "",
-        "allowPackageConflicts": "false",
-        "externalEndpoint": "",
-        "verbosityPush": "Detailed",
-        "searchPatternPack": "**/*.csproj",
-        "configurationToPack": "$(BuildConfiguration)",
-        "outputDir": "$(Build.ArtifactStagingDirectory)",
-        "versioningScheme": "off",
-        "includeReferencedProjects": "false",
-        "versionEnvVar": "",
-        "requestedMajorVersion": "1",
-        "requestedMinorVersion": "0",
-        "requestedPatchVersion": "0",
-        "packTimezone": "utc",
-        "includeSymbols": "false",
-        "toolPackage": "false",
-        "buildProperties": "",
-        "verbosityPack": "Detailed",
-        "arguments": "install Microsoft.DotNet.Build.Tasks.ScepOps -source https://devdiv.pkgs.visualstudio.com/_packaging/dotnet-core-internal-tooling/nuget/v3/index.json -OutputDirectory $(Build.StagingDirectory)"
-      }
-    },
     {
       "environment": {},
       "enabled": true,
       "continueOnError": true,
       "alwaysRun": false,
-      "displayName": "Uninstall SCEP",
+      "displayName": "Uninstall SCEP if script exists",
       "timeoutInMinutes": 0,
-      "refName": "Task52",
+      "condition": "succeededOrFailed()",
       "task": {
-        "id": "c6c4c611-aa2e-4a33-b606-5eaba2196824",
-        "versionSpec": "1.*",
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+        "versionSpec": "2.*",
         "definitionType": "task"
       },
       "inputs": {
-        "solution": "$(Build.StagingDirectory)\\Microsoft.DotNet.Build.Tasks.ScepOps.1.0.1\\build\\Microsoft.DotNet.Build.Tasks.ScepOps.targets",
-        "msbuildLocationMethod": "version",
-        "msbuildVersion": "14.0",
-        "msbuildArchitecture": "x64",
-        "msbuildLocation": "",
-        "platform": "",
-        "configuration": "",
-        "msbuildArguments": "/t:UninstallSCEP",
-        "clean": "false",
-        "maximumCpuCount": "false",
-        "restoreNugetPackages": "false",
-        "logProjectEvents": "false",
-        "createLogFile": "false"
+        "targetType": "inline",
+        "filePath": "",
+        "arguments": "-Action u -ToolsPath  $(ScepToolsPath)",
+        "script": "if (Test-Path \"$(ScepToolsPath)SCEP_Tools\\scripts\\Microsoft.DotNet.Build.Tasks.Extensions.ps1\") {\n     \"Microsoft.DotNet.Build.Tasks.Extensions.ps1 script found. Executing action...\"\n    & $(ScepToolsPath)SCEP_Tools\\scripts\\Microsoft.DotNet.Build.Tasks.Extensions.ps1 -Action u -ToolsPath $(ScepToolsPath)\n} else {\n   \"Microsoft.DotNet.Build.Tasks.Extensions.ps1 script does not exist. Moving on...\"\n}",
```

Comment:
Where is `$(ScepToolsPath)` defined?

Reply:
Huh, I guess got lost when pulling from upstream. Added in new commit